src/test/unit/controller/ContactTheTribunalFileController.test.ts

```diff
@@ -1,10 +1,18 @@
+import url from 'url';
+
 import ContactTheTribunalFileController from '../../../main/controllers/ContactTheTribunalFileController';
 import { getLanguageParam } from '../../../main/controllers/helpers/RouterHelpers';
+import * as routerHelpers from '../../../main/controllers/helpers/RouterHelpers';
 import { PageUrls } from '../../../main/definitions/constants';
 import { mockRequest } from '../mocks/mockRequest';
 import { mockResponse } from '../mocks/mockResponse';
 
 describe('Contact the tribunal file controller', () => {
+  const urlMock = {
+    host: 'http://localhost:3001',
```

Comment:
Does it matter to hardcode 3001 here if we're moving to/spread across cftlib (where its 3002)?

Also could this be declared elsewhere and imported? it looks like this is copy/pasted a ton of times

Reply:
It doesn't matter as we're matching the this value to a mocked request. I have moved this code out to a shared file and imported where its needed.